packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -59,6 +65,11 @@ const Basis = observer(
 
         const validateAmount = value => {
             const localized_message = <Localize i18n_default_text='Should not be 0 or empty' />;
+            const min_max_stake_message = (
+                <Localize
+                    i18n_default_text={`Stake must be between ${min_stake} ${currency} and ${max_stake} ${currency}`}
```

Comment:
Just curios: can we insert ${variables} in `<Localize />` like this? Because in most cases I see smth like this:
```
<Localize
                i18n_default_text='{{commission_percentage}}% of ( * {{multiplier}})'
                values={{
                    commission_percentage: Number((commission * 100) / (multiplier * amount)).toFixed(4),
                    multiplier,
                }}
 />
```

Reply:
@kate-deriv you made a good point. Unfortunately, Localize will not work with template string and variables inside it.
@henry-deriv please apply @kate-deriv's suggestion🙏🏻